test/xbgpu/test_engine.py

```diff
@@ -458,6 +458,11 @@ async def test_xengine_end_to_end(
         timestamp_step = n_samples_between_spectra * n_spectra_per_heap
         missing_antennas = set() if missing_antenna is None else {missing_antenna}
 
+        assert (
+            xbengine.sensors["chan-range"].value
+            == f"({n_channels_per_stream*CHANNEL_OFFSET},{n_channels_per_stream*CHANNEL_OFFSET + n_channels_per_stream - 1})"  # noqa: E501
```

Comment:
Rather than noqa'ing the extra-long line, I suggest pulling out variables for the first and last channel.